app/src/app/api/v0/threads/route.ts

```diff
@@ -0,0 +1,28 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+
+export const POST = apiHandler(async (req) => {
+  try {
+    const input: {
+      threadId: string;
+    } = await req.json();
+
+    if (!input.threadId || typeof input.threadId !== "string") {
+      return NextResponse.json(
+        { error: "Invalid threadId provided" },
+        { status: 400 },
+      );
+    }
+
+    const response = await db.models.AssistantThread.create({
+      assistantThreadId: input.threadId,
```

Comment:
I think you need to pass the assistant ID as well? Otherwise it will just be null?

Reply:
The workflow is as follows:
1. When the thread gets created, there is only the threadId. 
2. With the threadID in the Python script I extract all the messages and then later write the messages togehter with the assistantID to the table.
But actually we already have the assistantID in the environment variables so I could set it directly as well thats true.

Will update it